modules/nw-create-load-balancer-service.adoc

```diff
@@ -38,15 +38,22 @@ spec:
   - name: db
     port: 3306 <2>
   loadBalancerIP:
-  type: LoadBalancer <3>
+  loadBalancerSourceRanges: 10.0.0.0/8,192.168.0.0/16 <3>
+  type: LoadBalancer <4>
   selector:
-    name: mysql <4>
+    name: mysql <5>
 ----
 <1> Enter a descriptive name for the load balancer service.
 <2> Enter the same port that the service you want to expose is listening on.
-<3> Enter `loadbalancer` as the type.
-<4> Enter the name of the service.
-
+<3> Enter a list of specific IP addresses to restrict traffic through the load balancer. This field is ignored if the cloud-provider does not support the feature.
+<4> Enter `loadbalancer` as the type.
```

Comment:
```suggestion
<4> Enter `LoadBalancer` as the type.
```

Reply:
@Miciah I have addressed it in the latest commit.